ihttp/ihttp.go

```diff
@@ -57,28 +57,84 @@ func (s Handler) Handle(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 	host, port, _ := net.SplitHostPort(req.RemoteAddr)
-	s.Log = s.Log.WithValues("host", host, "port", port)
-	m := path.Dir(req.URL.Path)
-	if strings.HasPrefix(m, "/") {
-		m = trimFirstRune(path.Dir(req.URL.Path))
-	}
+	log := s.Log.WithValues("host", host, "port", port)
 	// If a mac address is provided, log it. Mac address is optional.
```

Comment:
I would rather see a comment here (and in the tftp code) that demonstrates the expected path.

In addition, renaming the variable to indicate that it is optional would that intent make that clear in multiple places (rather than a single comment that might get munged in the future).